compiler/src/iree/compiler/Dialect/Flow/Transforms/test/set_encoding.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-opt --iree-global-opt-set-encoding="pad-factor=16" --cse --split-input-file %s | FileCheck %s
+// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(util.func(iree-flow-set-encoding))" %s | FileCheck %s
```

Comment:
The default value of `pad-factor` is updated to 32 in https://github.com/iree-org/iree/pull/17931. And here we drop the `pad-factor=16` from the test, so they become 32.